### PR TITLE
docs: Changelog

### PR DESCRIPTION
**Pull request by frondeus**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
+# Common Data Layer Changelog
+
+## Unreleased 
+
+### Breaking changes
+Removed document-storage
+Removed blob-store
```
Comment (smores56) [inline CHANGELOG.md:7]: Archived?

Comment (frondeus) [reply CHANGELOG.md:7]: True that!

File: CHANGELOG.md [hunk 2]
```diff
@@ -0,0 +1,67 @@
+# Common Data Layer Changelog
+
+## Unreleased 
+
+### Breaking changes
+Removed document-storage
```
Comment (frondeus) [inline CHANGELOG.md:6]:
```suggestion
Archived document-storage crate
```

File: CHANGELOG.md (hunk 1 above)
Comment (frondeus) [inline CHANGELOG.md:7]:
```suggestion
Archived blob-store crate
```

**Review by luke-biel — APPROVED** [review]
You need te rebase with develop before you'll be able to merge this in

**Review by luke-biel — APPROVED** [review]

**Pull request merged**